bin/ch/ChakraRtInterface.cpp

```diff
@@ -122,6 +123,7 @@ bool ChakraRTInterface::LoadChakraDll(ArgInfo* argInfo, HINSTANCE *outLibrary)
     m_jsApiHooks.pfJsrtGetArrayBufferStorage = (JsAPIHooks::JsrtGetArrayBufferStoragePtr)GetChakraCoreSymbol(library, "JsGetArrayBufferStorage");
     m_jsApiHooks.pfJsrtHasException = (JsAPIHooks::JsrtHasExceptionPtr)GetChakraCoreSymbol(library, "JsHasException");
     m_jsApiHooks.pfJsrtSetException = (JsAPIHooks::JsrtSetExceptionPtr)GetChakraCoreSymbol(library, "JsSetException");
+    m_jsApiHooks.pfJsrtGetAndClearExceptionWithMetadata = (JsAPIHooks::JsrtGetAndClearExceptiopnWithMetadataPtr)GetChakraCoreSymbol(library, "JsGetAndClearExceptionWithMetadata");
```

Comment:
I don't know why ch doesn't just link to CC and call CC APIs directly but for whatever reason it wraps them with the ChakraRTInterface class - this call gets the JsGetAndClearExceptionWithMetadata function from CC, related edits in ChakraRtInterface.h  provide the pointer it will be stored in.

Reply:
Could this be another Chakra-full relic? @Penguinwizzard do you know anything about this?